src/dialogsearch.cpp

```diff
@@ -138,21 +138,19 @@ void DialogSearch::on_button_search_clicked()
                 std::vector<int> parameters;
                 std::tie(mapID, eventID, eventPage, sourceLine, parameters) = r;
 
-                std::vector<std::string> maps;
                 auto mm = mapID;
+                QStringList maps_rev;
                 do
                 {
-                    auto mapinfo = Data::treemap.maps[mm];
-                    maps.push_back(mapinfo.name);
+                    auto& mapinfo = Data::treemap.maps[mm];
+                    maps_rev << QString::fromStdString(mapinfo.name);
```

Comment:
This will reverse the order of the maps, I wanted them the other way around (from root to the map). Really change this? We could use a `std::stack` instead of a stringlist, but I like the `join` function of `QStringList`